packages/app/src/background/services/credentials/utils.ts

```diff
@@ -112,40 +112,36 @@ const verifiableCredentialSchema: yup.Schema<IVerifiableCredential> = yup.object
 
 /**
  * Serializes a CryptkeeperVerifiableCredential object into a JSON string.
- * @param cryptkeeperVerifiableCredential An object representing a CryptkeeperVerifiableCredential.
+ * @param cryptkeeperVC An object representing a CryptkeeperVerifiableCredential.
  * @returns A string representing a CryptkeeperVerifiableCredential.
  */
-export function serializeCryptkeeperVerifiableCredential(
-  cryptkeeperVerifiableCredential: ICryptkeeperVerifiableCredential,
-): string {
+export function serializeCryptkeeperVC(cryptkeeperVC: ICryptkeeperVerifiableCredential): string {
   return JSON.stringify({
-    verifiableCredential: serializeVerifiableCredential(cryptkeeperVerifiableCredential.verifiableCredential),
-    metadata: cryptkeeperVerifiableCredential.metadata,
+    verifiableCredential: serializeVC(cryptkeeperVC.verifiableCredential),
+    metadata: cryptkeeperVC.metadata,
   });
 }
 
 /**
  * Deserializes a CryptkeeperVerifiableCredential JSON string into a CryptkeeperVerifiableCredential object.
- * @param serializedCryptkeeperVerifiableCredential A JSON string representing a CryptkeeperVerifiableCredential.
+ * @param serializedCryptkeeperVC A JSON string representing a CryptkeeperVerifiableCredential.
  * @returns A CryptkeeperVerifiableCredential object. Throws error if the object is not a valid CryptkeeperVerifiableCredential.
  */
-export async function deserializeCryptkeeperVerifiableCredential(
-  serializedCryptkeeperVerifiableCredential: string,
+export async function deserializeCryptkeeperVC(
+  serializedCryptkeeperVC: string,
```

Comment:
It can be just serialized. You've already have the method name which tell what it is. This is related to all the cases in this file.